x-pack/plugins/security_solution/public/resolver/store/data/node_events_in_category_model.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { PanelViewAndParameters, NodeEventsInCategoryState } from '../../types';
+
+/**
+ * True if `nodeEventsInCategory` contains data that is relevant to `panelViewAndParameters`.
+ * This is used by the reducer to enforce that the nodeEventsInCategory state is always valid.
+ */
+export function isValid(
```

Comment:
❔ You're not really checking that they're `valid` so much as that they're `in sync` with each other here, right?

Reply:
Yea. A super long and more correct name could be:  `nodeEventsInCategoryContainsDataRelevantToPanelViewAndParameters`.

Reply:
Going with 'isRelevantToPanelViewAndParameters'